worlds/jakanddaxter/regs/BoggySwampRegions.py

```diff
@@ -135,23 +135,23 @@ def can_jump_higher(state: CollectionState, p: int) -> bool:
     fourth_tether.connect(last_tar_pit, rule=lambda state: can_jump_farther(state, player))
     fourth_tether.connect(main_area)  # Fall down.
 
-    multiworld.regions.append(main_area)
-    multiworld.regions.append(first_bats)
-    multiworld.regions.append(first_jump_pad)
-    multiworld.regions.append(first_tether)
-    multiworld.regions.append(first_tether_rat_colony)
-    multiworld.regions.append(second_jump_pad)
-    multiworld.regions.append(first_pole_course)
-    multiworld.regions.append(second_tether)
-    multiworld.regions.append(second_bats)
-    multiworld.regions.append(third_jump_pad)
-    multiworld.regions.append(fourth_jump_pad)
-    multiworld.regions.append(flut_flut_pad)
-    multiworld.regions.append(flut_flut_course)
-    multiworld.regions.append(farthy_snacks)
-    multiworld.regions.append(box_field)
-    multiworld.regions.append(last_tar_pit)
-    multiworld.regions.append(fourth_tether)
+    world.level_to_regions[level_name].append(main_area)
+    world.level_to_regions[level_name].append(first_bats)
+    world.level_to_regions[level_name].append(first_jump_pad)
+    world.level_to_regions[level_name].append(first_tether)
+    world.level_to_regions[level_name].append(first_tether_rat_colony)
+    world.level_to_regions[level_name].append(second_jump_pad)
+    world.level_to_regions[level_name].append(first_pole_course)
+    world.level_to_regions[level_name].append(second_tether)
+    world.level_to_regions[level_name].append(second_bats)
+    world.level_to_regions[level_name].append(third_jump_pad)
+    world.level_to_regions[level_name].append(fourth_jump_pad)
+    world.level_to_regions[level_name].append(flut_flut_pad)
+    world.level_to_regions[level_name].append(flut_flut_course)
+    world.level_to_regions[level_name].append(farthy_snacks)
+    world.level_to_regions[level_name].append(box_field)
+    world.level_to_regions[level_name].append(last_tar_pit)
+    world.level_to_regions[level_name].append(fourth_tether)
```

Comment:
optional ofc but :)
```suggestion
    world.level_to_regions[level_name].extend(
        main_area,
        first_bats,
        first_jump_pad,
        first_tether,
        first_tether_rat_colony,
        second_jump_pad,
        first_pole_course,
        second_tether,
        second_bats,
        third_jump_pad,
        fourth_jump_pad,
        flut_flut_pad,
        flut_flut_course,
        farthy_snacks,
        box_field,
        last_tar_pit,
        fourth_tether,
    )

Reply:
The result of a massive find and replace lol...